custom/bin/ARE/data/webservice/angular/envControl/component/help/faq/html/faq21en.html

```diff
@@ -1,10 +1,9 @@
 <p>
-    Zum Löschen bereits eingerichteter Befehle oder Geräte haben Sie 2 Möglichkeiten:
+    You have 2 options to delete commands or devices:
 <ol>
-    <li>Klick auf den Button „Löschen aktivieren“: Der nächste Button, der ausgewählt wird, wird gelöscht.
+    <li>1. Select the button “Activate Deleting” --> The next selected button is deleted.
```

Comment:
here you do not need numbers (1.) because `<ol>` does automatically generate them.